layouts/shortcodes/kubeflow-current-version.html

```diff
@@ -0,0 +1 @@
+0.5.1
```

Comment:
Ah, I see! Thanks for fixing the content. I have a question for @OfficePop and @joeliedtke: Is it the intention that this value refects the stable version or the latest version of Kubeflow? FYI the master version of the docs always reflects the latest version of Kubeflow, unless otherwise stated in the docs.

Another thing to check is whether you need `0.5.1` or `v0.5.1` for the purposes of your doc. (I haven't delved into the doc itself.)

The stable version is currently 0.5.1, and we usually refer to it as "v0.5.1". We already have a shortcode for this version, although it's out of date. I've raised PR https://github.com/kubeflow/website/pull/933 to update it. The shortcode is at `layouts/shortcodes/kf-stable-tag.html`.

If you're looking for the latest version, I've coincidentally just reviewed and commented in PR https://github.com/kubeflow/website/pull/928, which also needs the latest version. Since that PR and potentially your PR both need this shortcode, I've gone ahead and created the new shortcode in my own PR https://github.com/kubeflow/website/pull/933.

See the [Kubeflow releases](https://github.com/kubeflow/kubeflow/releases).

Reply:
>  Is it the intention that this value refects the stable version or the latest version of Kubeflow?

@sarahmaddox yes, that is goal. We were trying to fix the fact that the live version of the page currently shows an older version (v0.4.1 I believe). Based on your comment I've removed the kubeflow current version tag and added the kf-stable-tag shortlink as well, so all should be well when both PRs are merged!